src/core/buildFunctions.js

```diff
@@ -574,19 +577,47 @@ function computeTriangleBounds( geo, fullBounds ) {
 
 	}
 
+	// used for normalized positions
+	const getters = [ 'getX', 'getY', 'getZ' ];
+
 	for ( let tri = 0; tri < triCount; tri ++ ) {
 
 		const tri3 = tri * 3;
 		const tri6 = tri * 6;
-		const ai = index[ tri3 + 0 ] * stride + bufferOffset;
-		const bi = index[ tri3 + 1 ] * stride + bufferOffset;
-		const ci = index[ tri3 + 2 ] * stride + bufferOffset;
+
+		let ai, bi, ci;
+
+		if ( normalized ) {
+
+			ai = index[ tri3 + 0 ];
+			bi = index[ tri3 + 1 ];
+			ci = index[ tri3 + 2 ];
+
+		} else {
+
+			ai = index[ tri3 + 0 ] * stride + bufferOffset;
+			bi = index[ tri3 + 1 ] * stride + bufferOffset;
+			ci = index[ tri3 + 2 ] * stride + bufferOffset;
+
+		}
```

Comment:
Sorry I missed this - why is this condition necessary? There shouldn't be any changes relating to the index buffer, right?

Reply:
When we use the getters, `ai`, `bi`, and `ci` should directly reference the index, but when we access from the array we also need to take into account the stride and buffer offset.

Though we can probably avoid this case by setting `stride` to 1 and `bufferOffset` to 0 when we're dealing with a normalized attribute.

Reply:
Alternatively, we can in the if statement below, add in the stride and bufferOffset to the expression to find the correct value in `posArr`, rather than putting them in `ai` and co here.